### ISSUE TITLE
Verify correctness of the required browser features pages

### ISSUE THREAD

**Issue by timvandermeij** [mozilla/pdf.js#11954]
On https://github.com/mozilla/pdf.js/wiki/Required-Browser-Features we mention required browser features for PDF.js, but I have the feeling that not all of them are necessary nowadays since we moved to using more modern features over the years. Moreover, as indicated in https://github.com/mozilla/pdf.js/issues/11950#issuecomment-637149085, some features may be missing such as range requests or `async/await`.

In short, we should verify which are still necessary and update the wiki page and the automated feature detection if possible.

**Comment by timvandermeij** [comment]
There is also the idea to replace the supported browsers overview with a verification based on which demo works; see https://github.com/mozilla/pdf.js/pull/12170#issuecomment-687562858. If we use that, we might also consider dropping the required browser features page since I think it's not relevant then anymore.

**Comment by Snuffleupagus** [comment]
Given that this page hasn't received any *meaningful* updates since 2014[1], and that first of all fixing it and secondly keeping it up-to-date will take quite a bit of effort, I'd suggest that we move forward with https://github.com/mozilla/pdf.js/issues/11954#issuecomment-687587562 instead.

---
[1] There's been changes since then, but that's when the last feature-test was added.
